### PR TITLE
fix problem of message

### PR DESCRIPTION
**Pull request by Sinyu104**
### Change a few things in MessageServelt let remove html tag.
image

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 1]
```diff
@@ -90,12 +90,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
         String user = userService.getCurrentUser().getEmail();
 
-        String text = request.getParameter("text");
+        String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
         Whitelist whitelist = Whitelist.basicWithImages();
         whitelist.addTags( "h1", "h2", "h3", "h4", "h5", "h6");
         String userText = Jsoup.clean(text, whitelist);
 
-       // System.out.println(userText);
+       System.out.println(userText);
```
Comment (Sinyu104) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java]: This line is only for debugging.
I think I should delete this line.

Comment (nicolercl) [reply src/main/java/com/google/codeu/servlets/MessageServlet.java]: Sure.

Comment (Sinyu104) [reply src/main/java/com/google/codeu/servlets/MessageServlet.java]: # Delete !!
@nicolercl

**Review by nicolercl — APPROVED** [review]

**Comment by Sinyu104** [comment]
Thanks!!

**Pull request merged**